bin/cron/teacher_applications_to_s3.rb

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env ruby
+require_relative '../../dashboard/config/environment'
+require 'cdo/aws/s3'
+require 'csv'
+
+def main
+  applications = Pd::Application::Teacher1819Application.left_outer_joins(:regional_partner).pluck(:course, :name, :locked_at, :status)
```

Comment:
I think the Style guideline recommends breaking up lines longer than 80 characters like so:

``` ruby
applications = Pd::Application::Teacher1819Application
  .left_outer_joins(:regional_partner)
  .pluck(:course, :name, :locked_at, :status)
```

Reply:
Done, thanks!